pyiron/base/generic/parameters.py

```diff
@@ -980,3 +980,11 @@ def _find_line(self, key_name):
                                                               self._dataset["Value"][i]))
             error_msg = "\n".join(error_msg)
             raise ValueError(error_msg)
+
+    def clear_all(self):
```

Comment:
Can not you use `load_default()` ?

Reply:
In this case yes. But what if `load_default()` does not give an empty dataset. In that case, wouldn't `clear_all` be useful?

Reply:
You are right.